addons/pos_epson_printer/static/src/js/printers.js

```diff
@@ -24,7 +24,9 @@ var EpsonPrinter = core.Class.extend(PrinterMixin, {
         } else {
             Gui.showPopup('ErrorPopup', {
                 'title': _t('Connection to the printer failed'),
-                'body':  _t('Please check if the printer is still connected, if the configured IP address is correct and if your printer supports the ePOS protocol.'),
+                'body':  _t('Please check if the printer is still connected, if the configured IP address is correct and if your printer supports the ePOS protocol.') +
+                    _t("Some browsers don't allow HTTP calls from websites to devices in the network (for security reasons). If it is the case, you will need to follow Odoo's documentation for" +
```

Comment:
Don't need a new `_t() `you can make your message in only one.
Your need escape your `'` and not make `" ' "`
Make some `newline` in your message it is not very `lisible`

Reply:
@qle-odoo ok I did merge the text and escaped the `'` characters as you asked (just struggle to push it).


However, for the `newline` part, I would like to do that too. But I can't understand how to do so :/ It looks like `\n` should do the trick (like in https://github.com/odoo/odoo/blob/d3a8bdb391935158b1b2985e868448f5a31c47af/addons/point_of_sale/static/src/js/Screens/PaymentScreen/PaymentScreen.js#L284 ) but in practice it does not, see:
![image](https://user-images.githubusercontent.com/60775325/172845810-a3d05ef1-dbea-46cc-81cf-6880ef12852c.png)

@caburj maybe you do have an idea on the matter ?